src/openrct2/ride/VehicleSubpositionData.cpp

```diff
@@ -21435,6 +21435,2376 @@ CREATE_VEHICLE_INFO(TrackVehicleInfo_9571BC, {
+    {48, 20, 0, 16, 6, 12}, {49, 20, -1, 16, 6, 12}, {50, 20, -2, 16, 6, 12}, {51, 20, -3, 16, 6, 12}, {52, 20, -3, 16, 6, 12}, 
+    {52, 20, -4, 16, 6, 12}, {53, 20, -5, 16, 6, 12}, {54, 20, -6, 16, 7, 11}, {55, 20, -7, 16, 7, 11}, {56, 20, -8, 16, 7, 11}, 
+    {57, 20, -9, 16, 7, 11}, {58, 20, -10, 16, 7, 11}, {59, 20, -11, 16, 7, 11}, {60, 20, -12, 16, 7, 11}, {61, 20, -13, 16, 7, 11}, 
+    {62, 20, -14, 16, 7, 11}, {62, 20, -15, 16, 7, 11}, {63, 20, -16, 16, 7, 11}, {64, 20, -16, 16, 7, 11}, {65, 20, -17, 16, 7, 11}, 
+    {66, 20, -18, 16, 7, 11}, {67, 20, -19, 16, 7, 11}, {67, 20, -20, 16, 7, 11}, {68, 20, -21, 16, 7, 11}, {69, 20, -23, 16, 7, 11}, 
+    {70, 20, -24, 16, 7, 11}, {71, 20, -25, 16, 7, 10}, {72, 20, -26, 16, 7, 10}, {72, 20, -27, 16, 7, 10}, {73, 20, -28, 16, 7, 10}, 
+    {74, 20, -29, 16, 7, 10}, {75, 20, -30, 16, 7, 10}, {76, 20, -31, 16, 7, 10}, {76, 20, -32, 16, 7, 10}, {77, 20, -33, 16, 7, 10}, 
+    {78, 20, -34, 16, 7, 10}, {79, 20, -35, 16, 7, 10}, {79, 20, -36, 16, 7, 10}, {80, 20, -37, 16, 7, 10}, {81, 20, -38, 16, 7, 10}, 
+    {82, 20, -39, 16, 7, 10}, {82, 20, -40, 16, 7, 10}, {83, 20, -42, 16, 7, 10}, {84, 20, -43, 16, 7, 4}, {85, 20, -44, 16, 7, 4}, 
+    {85, 20, -45, 16, 7, 4}, {86, 19, -46, 16, 7, 4}, {87, 19, -47, 16, 7, 4}, {87, 19, -48, 16, 7, 4}, {88, 19, -49, 16, 7, 4}, 
+    {89, 19, -50, 16, 7, 4}, {90, 19, -51, 16, 7, 4}, {90, 19, -53, 16, 7, 4}, {91, 19, -54, 16, 7, 4}, {92, 19, -55, 16, 7, 4}, 
+    {92, 19, -56, 16, 7, 4}, {93, 19, -57, 16, 7, 4}, {94, 19, -58, 16, 7, 4}, {95, 19, -59, 16, 7, 4}, {95, 19, -60, 16, 7, 4}, 
+    {96, 19, -61, 16, 8, 3}, {97, 19, -63, 16, 8, 3}, {97, 18, -64, 16, 8, 3}, {98, 18, -65, 16, 8, 3}, {99, 18, -66, 16, 8, 3}, 
+    {99, 18, -67, 16, 8, 3}, {100, 18, -68, 16, 8, 3}, {101, 18, -69, 16, 8, 3}, {101, 18, -70, 16, 8, 3}, {102, 18, -72, 16, 8, 3}, 
+    {103, 18, -73, 16, 8, 3}, {103, 18, -74, 16, 8, 3}, {104, 18, -75, 16, 8, 3}, {105, 18, -76, 16, 8, 3}, {105, 18, -77, 16, 8, 3}, 
+    {106, 18, -78, 16, 8, 3}, {106, 18, -79, 16, 8, 3}, {107, 18, -80, 16, 8, 3}, {108, 17, -82, 16, 8, 3}, {108, 17, -83, 16, 8, 3}, 
+    {109, 17, -84, 16, 8, 3}, {110, 17, -85, 16, 8, 3}, {110, 17, -86, 16, 8, 3}, {111, 17, -87, 16, 8, 3}, {111, 17, -88, 16, 8, 3}, 
+    {112, 17, -89, 16, 8, 3}, {113, 17, -91, 16, 8, 3}, {113, 17, -92, 16, 8, 3}, {114, 17, -93, 16, 8, 3}, {115, 17, -94, 16, 8, 3}, 
+    {115, 17, -95, 16, 8, 0}, {116, 17, -96, 16, 8, 0}, {116, 17, -97, 16, 8, 0}, {117, 17, -98, 16, 8, 0}, {117, 17, -99, 16, 8, 0}, 
+    {118, 17, -100, 16, 8, 0}, {119, 17, -102, 16, 8, 0}, {119, 17, -103, 16, 8, 0}, {120, 16, -104, 16, 8, 0}, {120, 16, -105, 16, 8, 0}, 
+    {121, 16, -106, 16, 8, 0}, {121, 16, -107, 16, 8, 0}, {122, 16, -108, 16, 8, 0}, {123, 16, -109, 16, 8, 0}, {123, 16, -110, 16, 8, 0}, 
+    {124, 16, -111, 16, 8, 0}, {124, 16, -112, 16, 8, 0}, {125, 16, -113, 16, 8, 0}, {125, 16, -114, 16, 8, 0}, {126, 1
```

Comment:
This file needs the regex applied.

Reply:
Which regex?

Reply:
https://github.com/OpenRCT2/OpenRCT2/pull/16164#issuecomment-1191148643

@Broxzier would you be able to format this for X7? Might be an idea to add a comment to the file with the regex.

Reply:
Sure, I can do that this evening. The comment can link to the public gist I made, rexed is there at the bottom.

https://gist.github.com/Broxzier/03041c723d1b23d9468ff4955ce3b659?permalink_comment_id=4239992#gistcomment-4239992